deployer/commands/generate/resource_allocation/daemonset_requests.yaml

```diff
@@ -40,7 +40,7 @@ gke:
   #
   2i2c:
     requesting_daemon_sets: calico-node,fluentbit-gke,gke-metadata-server,gke-metrics-agent,ip-masq-agent,netd,pdcsi-node,support-cryptnono,support-prometheus-node-exporter
-    other_daemon_sets: binder-staging-dind,binder-staging-image-cleaner,imagebuilding-demo-binderhub-service-docker-api
```

Comment:
I think the last two may still be running here?

Reply:
Ah, good catch. The file was actually updated automatically by the deployer command: `deployer generate resource-allocation daemonset-requests 2i2c` and I think it removed the `imagebuilding-demo-binderhub-service-docker-api` because it was not running.

I fired an image build on the imagebuilding-demo hub and ran the command again and it picked it up. I don't think this command is documented though so not sure where to capture this shortcoming. (cc @consideRatio as tbe

Reply:
For things running on the user nodes only, when there isn't a user node, this could happen.

Currently this is only used to provide information that is manually considered. I suggest doing nothing for now.